servers/src/main/java/tachyon/master/MasterSource.java

```diff
@@ -160,43 +195,100 @@ public MetricRegistry getMetricRegistry() {
     return mMetricRegistry;
   }
 
-  public void incCreateFileOps() {
-    mCreateFileOps.inc();
+  public void incCompleteFileOps() {
+    mCompleteFileOps.inc();
+  }
+
+  public void incFilesCompleted() {
+    mFilesCompleted.inc();
+  }
+
+  public void incReleaseFileOps() {
+    mReleaseFileOps.inc();
+
+  }
+
+  public void incFilesReleased(long freeInodes) {
+    mFilesReleased.inc(freeInodes);
+  }
+
+  public void incCreatePathOps() {
+    mCreatePathOps.inc();
+  }
+
+  public void incDeletePathOps() {
+    mDeletePathOps.inc();
+  }
+
+  public void incPathsCreated(long n) {
+    mPathsCreated.inc(n);
   }
 
-  public void incDeleteFileOps() {
-    mDeleteFileOps.inc();
+  public void incDirectoriesCreated() {
+    mDirectoriesCreated.inc();
   }
 
-  public void incFilesCreated() {
-    mFilesCreated.inc();
+  public void incCreateDirectoriesOps() {
+    mCreateDirectoryOps.inc();
   }
 
-  public void incFilesCreated(long n) {
-    mFilesCreated.inc(n);
+  public void incPathsDeleted(long n) {
+    mPathsDeleted.inc(n);
   }
 
-  public void incFilesDeleted() {
-    mFilesDeleted.inc();
+  public void incPathsRenamed() {
+    mPathsRenamed.inc();
   }
 
-  public void incFilesDeleted(long n) {
-    mFilesDeleted.inc(n);
+  public void incFilesPersisted() {
+    mFilesPersisted.inc();
   }
 
-  public void incFilesRenamed() {
-    mFilesRenamed.inc();
+  public void incGetFileInfoOps() {
+    mGetFileInfoOps.inc();
   }
 
-  public void incFilesCheckpointed() {
-    mFilesCheckpointed.inc();
+  public void incFileInfosGot(long n) {
+    mFileInfosGot.inc(n);
   }
 
-  public void incGetFileStatusOps() {
-    mGetFileStatusOps.inc();
+  public void incGetFileBlockInfoOps() {
+    mGetFileBlockInfoOps.inc();
+  }
+
+  public void incFileBlockInfosGot(long n) {
+    mFileBlockInfosGot.inc(n);
   }
 
   public void incRenameOps() {
     mRenameOps.inc();
   }
+
+  public void incPathUnmounted() {
+    mPathsUnmounted.inc();
+  }
+
+  public void incUnmountPathsOps() {
+    mUnmountPathsOps.inc();
+  }
+
+  public void incPathMounted() {
+    mPathsMounted.inc();
+  }
+
+  public void incMountPathsOps() {
+    mMountPathsOps.inc();
+  }
+
+  public void incSetStatesOps() {
```

Comment:
The method this is tracking is setState(), so this should be called incSetStateOps()?

Also, it is confusing because it is updating a variable mSetFileStatusOps, and its name is: SetFileStatusOps. Also, the documentation uses a different name.